client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.js

```diff
@@ -1,15 +1,70 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
+import { Button } from 'reactstrap';
+import { LoadingSpinner, Message } from '_common';
 import DataFilesListing from '../DataFilesListing/DataFilesListing';
+import './DataFilesProjectFileListing.module.scss';
 
 const DataFilesProjectFileListing = ({ system, path }) => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch({
+      type: 'PROJECTS_GET_METADATA',
+      payload: system
+    });
+  }, [system]);
+
+  const metadata = useSelector(state => state.projects.metadata);
+
+  const onEdit = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'editproject', props: {} }
+    });
+  };
+
+  const onManage = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'manageproject', props: {} }
+    });
+  };
+
+  if (metadata.loading) {
+    return <LoadingSpinner />;
+  }
+
+  if (metadata.error) {
+    return (
+      <Message type="warn">
+        We were unable to retrieve this shared workspace.
+      </Message>
+    );
+  }
+
   return (
-    <DataFilesListing
-      api="tapis"
-      scheme="projects"
-      system={system}
-      path={path || '/'}
-    />
+    <div styleName="root">
+      <div styleName="title-bar">
+        <h6>{metadata.title}</h6>
+        <div styleName="controls">
+          <Button color="link" styleName="edit" onClick={onEdit}>
+            <h6>Edit Descriptions</h6>
+          </Button>
+          <span styleName="separator">|</span>
+          <Button color="link" styleName="edit" onClick={onManage}>
+            <h6>Manage Team</h6>
+          </Button>
+        </div>
+      </div>
+      <div styleName="description">{metadata.description}</div>
```

Comment:
When navigating into a folder, or reloading the projects files listing, this all disappears
![Screen Shot 2020-12-11 at 1 29 42 AM](https://user-images.githubusercontent.com/20326896/101875517-600fc200-3b50-11eb-81e1-9a50e9ff1adf.png)

Reply:
Fixed by changing projects scheme to /tapis/projects/:system/:path